aksel.nav.no/website/sanity/schema/util.ts

```diff
@@ -31,6 +31,7 @@ export function sanitizeSlug(input: string) {
       })
       // Replace accented characters with non-accented equivalents
       .normalize("NFD")
+      // biome-ignore lint/suspicious/noMisleadingCharacterClass: Nextjs-loader (webpack/loaders/next-swc-loader.js) does not support adding `v` to regex expressions
```

Comment:
I suggest mentioning that it's a flag
```suggestion
      // biome-ignore lint/suspicious/noMisleadingCharacterClass: Nextjs-loader (webpack/loaders/next-swc-loader.js) does not support the `v` flag in regex expressions
```